### DISCUSSION TITLE
Multiple Suites with Single Container

### DISCUSSION THREAD

**Discussion by AlexWeinstein92** [testcontainers/testcontainers-java#6929] [Q&A]
Currently I have a batch of Suites that inherit from `TestContainersForAll` through a common custom trait. I want them all to use the same container but cannot seem to figure it out.

Currently my issue is that things compile fine but each Suite only takes in `containerDef` override as a parameter. This would seem to imply that they each spin up their own container, which is not what I want. Accordingly the containers have many errors like `failed to close the response` and ` Local Docker Compose exited abnormally with code 1 whilst running command: --compatibility up -d`.

Wondering if there is something else I should use other than `TestContainersForAll`. I am using `override def nestedSuites` to run the suites and passing in my `containerDef` as a parameter to each to override the one spun up in my custom trait that uses `TestContainersForAll`.

**Comment by eddumelendez** [comment]
I think this belongs to https://github.com/testcontainers/testcontainers-scala
